edb/pgsql/metaschema.py

```diff
@@ -3117,6 +3117,7 @@ def __init__(self) -> None:
                 'postgres environment variable',
                 'postgres configuration file',
                 'environment variable',
+                'configuration file',
```

Comment:
Would it be basically workable to just pretend it is an `environment variable`?

Patching this type to add the new field is kind of hairy, since you can't reorder enum values in postgres.

Reply:
I see, let me try!

Reply:
No, because now we have in-place upgrades, unfortunately, so we would want to just always do it this way, unfortunately

Reply:
Roger; I'll test and commit this diff.